bin/sync_cms_to_plugins_repo.js

```diff
@@ -11,6 +11,6 @@ console.log('Checking for CMS updates...')
 console.log('Changes to synchronize', changes)
 console.log('Synchronizing changes to plugins repo...')
 const updatedPlugins = updatePlugins(changes, plugins)
-fs.writeFile('site/plugins.json', JSON.stringify(updatedPlugins, null, 2))
+fs.writeFile('site/plugins.json', `${JSON.stringify(updatedPlugins, null, 2)}\n`)
```

Comment:
Adds a new line that was previously getting removed when doing the CMS to repo sync.